### PR TITLE
Cirrus: Use F33 VM image

### PR DESCRIPTION
**Pull request by cevich**
Depends on #8231

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
Tests don't look too hip @cevich

**Comment by cevich** [comment]
> Tests don't look too hip @cevich

Sorry...converted to a draft.

**Comment by cevich** [comment]
I'm hitting a major snafu with [the latest Fedora 33beta container images](https://github.com/containers/automation_images/pull/30) from containers/automation_images and could really use your help.  The VM images don't exhibit this problem, and I was able to narrow the issue down to a call in `/etc/profile`:

https://cirrus-ci.com/task/5765609151528960?command=setup#L18

If I run this manually, the error message returned is even more concerning:

```
$ podman run -it --rm quay.io/libpod/fedora_podman:c6039369259679744
[root@6d3d4678c740 podman]# less /etc/profile
[root@6d3d4678c740 podman]# /usr/bin/hostnamectl --transient
System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to create bus connection: Host is down
exit(1)
[root@6d3d4678c740 podman]# cat /etc/hostname
6d3d4678c740
```

For F32 and prior, `/etc/profile` used a call to `hostname` instead, which (of course) runs without any problems.

File: test/e2e/pod_infra_container_test.go [hunk 1]
```diff
@@ -383,12 +383,14 @@ var _ = Describe("Podman pod create", func() {
 		podID := session.OutputToString()
 
 		// verify we can add a host to the infra's /etc/hosts
-		session = podmanTest.Podman([]string{"run", "--pod", podID, "--add-host", "foobar:127.0.0.1", BB, "ping", "-c", "1", "foobar"})
+		// N/B: Using alpine for ping, since BB ping throws
+		//      permission denied error as of Fedora 33.
```
Comment (cevich) [inline test/e2e/pod_infra_container_test.go:387]: blah, that's kinda confusing.  Should say "...as of Fedora 33 seccomp and sysctl changes"

**Comment by cevich** [comment]
@rhatdan @mheon @baude when you have a moment PTAL (not urgent).  *Note:* There's a new problem with `Static Build` I will open an issue on, for now it's marked as "ignore-failure".

**Comment by rhatdan** [comment]
/approve
lgtm
@containers/podman-maintainers PTAL

**Comment by mheon** [comment]
LGTM

**Comment by edsantiago** [comment]
Is it absolutely necessary to cram the artifacts change into this PR?

**Comment by umohnani8** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by cevich** [comment]
> Is it absolutely necessary to cram the artifacts change into this PR?

It was a "while I'm at it" thing, having hit two "failure to upload" flakes today.  So we'd want to get it in ASAP anyway.

**Comment by edsantiago** [comment]
"while I'm at it" things are great; but when they're longer and more complex than the PR itself (at least what the PR title and git commit message show), a future maintainer will appreciate having them split out into separate PRs. Just a recommendation for next time.

**Pull request merged**